plugins/statistics.koplugin/main.lua

```diff
@@ -486,6 +490,7 @@ function ReaderStatistics:saveSettings(fields)
 end
 
 function ReaderStatistics:onReadSettings(config)
+    -- delay initialization for accurate total pages count of the doc
```

Comment:
Out of curious, what is setting the accurate total pages count? I wonder if we can emit an event after that operation.